lib/config/folderconfiguration.go

```diff
@@ -164,6 +166,11 @@ func (f *FolderConfiguration) prepare() {
 		f.RescanIntervalS = 0
 	}
 
+	if f.NotifyDelayS <= 0 {
+		f.FsNotifications = false
```

Comment:
This was the intention. No delay isn't possible, but it could arguably be set to the smallest possible value (1 second) when set to 0.